files/build_templates/docker_image_ctl.j2

```diff
@@ -644,6 +644,7 @@ start() {
         -v /var/run/hw-management:/var/run/hw-management:rw \
         -v mlnx_sdk_socket:/var/run/sx_sdk \
         -v /tmp/nv-syncd-shared/:/tmp \
+        -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket \
```

Comment:
>system_bus_socket [](http://example.com/codeflow?start=57&length=17)

Is it true that code inside pmon can manipulate any host service?

Reply:
yes, PMON can control host services. We need this creation/removal of the midplane interfaces

Reply:
@gpunathilell looks to be a hacky approach to me accessing host from a docker. Instead,
rshim can monitor the udev event for the pcie link to be up when the DPU is up and create the interface once the pcie link is up

Reply:
@prgeor DPU state is controlled by the PMON. PMON knows when to create or remove the midplane interface (rshim). Rshim can't subscribe to the event because the dependencies are opposite. Before removing the PCI interface, we need to stop rshim.

Reply:
@prgeor, @qiluo-msft, we will use the rshim service to control the midplane interface only in 202411 release. In 202505 it will be repleased with the physical function interface and the dependency to rshim and host services will be removed.

Reply:
@oleksandrivantsiv To reduce the attacking surface, could you limit the changes to only mellanox and only SmartSwitch?

Reply:
@qiluo-msft please check updated implementation

Reply:
@qiluo-msft kindly reminder